lib/node_modules/@stdlib/blas/ext/base/gnansumkbn2/test/test.main.js

```diff
@@ -151,14 +151,26 @@ tape( 'the function supports a negative `stride` parameter', function test( t )
 	t.end();
```

Comment:
@headlessNode Would you mind updating to remove the use of `floor` in this file and the other test file?